DependencyInjection/Configuration.php

```diff
@@ -298,6 +298,22 @@ private function createClientPluginNode()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('add_path')
+                    ->canBeEnabled()
+                    ->addDefaultsIfNotSet()
+                    ->info('Add a base path to the request.')
+                    ->children()
+                        ->scalarNode('path')
+                            ->info('Path to be added, e.g. /api/v1')
+                            ->isRequired()
+                            ->cannotBeEmpty()
+                        ->end()
+                        ->scalarNode('replace')
```

Comment:
are you sure about this one? looking at https://github.com/php-http/client-common/blob/master/src/Plugin/AddPathPlugin.php i think there is only the uri argument.

a request always has a path, so i think for path this just always prepends a path when configured, and there is no `replace` argument.